src/myrtlespeech/data/preprocess.py

```diff
@@ -222,3 +225,75 @@ def __repr__(self) -> str:
             + f" n_feature_masks={self.n_feature_masks},"
             + f" n_time_masks={self.n_time_masks})"
         )
+
+
+class MFCCLegacy:
+    """Legacy MFCC implementation using ``python_speech_features``.
+
+    This is required for backwards compatibility: **new users should not use
+    this class.** This class has the same API as
+    :py:class`torchaudio.transforms.MFCC`.
+
+    Args:
+        n_mfcc: Number of MFCC coefficients.
+
+        melkwargs: Dict containing mel-spectogram kwargs:
+
+            win_length:
+                Window length (in samples not ms.)
+
+            hop_length:
+                Stride length (in samples not ms.)
+
+        sample_rate: audio sample rate.
+    """
+
+    def __init__(self, n_mfcc, melkwargs, sample_rate=16000):
```

Comment:
```suggestion
    def __init__(self, n_mfcc: int, melkwargs: Optional[Dict] = None, sample_rate: int = 16000):
```

Reply:
`melkwargs` could be a `TypedDict` if it only takes two parameters.

Reply:
Thanks - updated to match suggestion (except that melkwargs isn't Optional).

`melkwargs` has a much larger number of possible parameters although just two are present in the current MFCC protobuf. We are unlikely to use any of the other args as this is a legacy def but just in case, I'm leaving as `Dict` instead of `TypedDict`.